vms/avm/txs/tx.go

```diff
@@ -47,7 +47,7 @@ type Tx struct {
 	Unsigned UnsignedTx          `serialize:"true" json:"unsignedTx"`
 	Creds    []*fxs.FxCredential `serialize:"true" json:"credentials"` // The credentials of this transaction
 
-	id    ids.ID
+	MyID  ids.ID `json:"id"`
```

Comment:
Just confirming, this is the only place we need to add this for all AVM transaction types because all `UnsignedTx` types are wrapped here?

Reply:
yup